src/services/services.js

```diff
@@ -704,23 +704,13 @@ export const SERVICES = {
 
             let query, variables;
 
-            if (coordinates && coordinates.length === 2) {
-                query = `  ${fragmentView}
-                       query ByLocation($site: String!, $originalSource: String, $coordinates: [Float]!, $filteredEdges: [String]!, $langCode: String!, $limit: Int!, $offset: Int!, $fromDate: String!, $toDate: String!, $sourceFilter: [String], $fulltextTerm: String) { 
```

Comment:
Let me merge this and then I'll have to revert this and #11 which already got merged.